src/lib/containers/TotalQueryResults.tsx

```diff
@@ -238,22 +291,47 @@ const TotalQueryResults: FunctionComponent<TotalQueryResultsProps> = ({
     executeQueryRequest!(initQueryRequest)
   }
 
-  const searchSelectionCriteriaPill = lastQueryRequest?.query.additionalFilters?.map(
-    el => {
-      const { columnName } = el
-      return el.values.map(value => {
-        return (
-          <SelectionCriteriaPill
-            key={value}
-            filter={{
+  const searchSelectionCriteriaPillProps:
+    | (SelectionCriteriaPillProps & {
+        key: string
+      })[]
```

Comment:
Instead of rendering an array of elements, I decided to compute an array of props and then render the elements inline (we render facet filters inline). I might revert this, I don't think I actually like this.